pkg/commands/image/create_test.go

```diff
@@ -5,6 +5,7 @@ package image_test
 
 import (
 	"encoding/json"
+	registryfakes "github.com/vmware-tanzu/kpack-cli/pkg/registry/fakes"
```

Comment:
nit: how'd this import get moved up here

Reply:
Fixed to move it back to where it belongs :D